src/settings.js

```diff
@@ -43,10 +43,27 @@ const list_options = ["position", "weight", "both", "none"];
  * @property {boolean} [showMakerCode] - if maker codes should be marked as such
  */
 
+// To try to
+let settingsJson;
+try {
+  settingsJson = fs.readFileSync("settings/settings.json", {
+    encoding: "utf8",
+  });
+} catch (err) {
+  if (err.code === "ENOENT") {
+    settingsJson = fs.readFileSync("settings.json", { encoding: "utf8" });
```

Comment:
This reminds me that an even older version of the queue had `settings.js` in the root of the queue.
I wonder if we could add migration code that loads `settings.js` if neither `settings/settings.json` nor `settings.json` are present (in a separate issue/PR), just to make it way easier for people to upgrade.

Reply:
I think that's probably asking for trouble. Converting one JSON format to another is one thing, but trying to parse and convert an actual code file is... it would work, but I don't think it's necessarily *advisable*.

Might be worth thinking more about, though probably not as part of this.

Reply:
Created the issue https://github.com/fluid-queue/fluid-queue/issues/43.